app/globals.css

```diff
@@ -175,6 +176,24 @@
   --shadow-2xl: 0 1px 3px 0px oklch(0.00 0 0 / 0.25);
 }
 
+@font-face {
+  font-family: 'Labora Rounded';
+  src: url('/fonts/LaboraRounded-Regular.woff2') format('woff2'),
```

Comment:
I don't see this font being added. Also, why not use one from google fonts instead?